Cargo.toml

```diff
@@ -1,2 +1,2 @@
 [workspace]
-members = ["cnd", "btsieve", "internal/*"]
```

Comment:
:question: Doesn't `cargo-tomlfmt` enforce alphabetical order?

Reply:
Nope! (I just tested)

Reply:
It looks like the toml specification [doesn't guarantee order for items in tables](https://github.com/toml-lang/toml#table).